LINQ/Task1/LinqTask.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using Task1.DoNotChange;
+
+namespace Task1
+{
+    public static class LinqTask
+    {
+        public static IEnumerable<Customer> Linq1(IEnumerable<Customer> customers, decimal limit)
+        {
+            return customers.Where(x => x.Orders.Select(x => x.Total).Sum() > limit);
```

Comment:
be careful of possible multiple enumerations of IEnumerable. My suggestion is to use ".ToList()" when using enumerations:
To learn more: multiple enumerations of IEnumerable:
 - https://www.jetbrains.com/help/resharper/PossibleMultipleEnumeration.html
 - https://itecnote.com/tecnote/c-handling-warning-for-possible-multiple-enumeration-of-ienumerable/
 -